cli/secrets.py

```diff
@@ -0,0 +1,19 @@
+import boto3
+
+
+def get_secret(secret_id, region_name="eu-west-1"):
+    sts_client = boto3.client("sts")
```

Comment:
We should be creating 1 session per invocation of the whole CLI - pattern elsewhere is that functions which use AWS services have signatures like 
```python
def get_secret(session, *, secret_id)
```